src/sentry/search/utils.py

```diff
@@ -42,18 +45,18 @@ def get_user_tag(projects, key, value):
         return f"{key}:{value}"
     except DataError:
         raise InvalidQuery(f"malformed '{key}:' query '{value}'.")
-    return euser.tag_value
+    return euser.tag_value  # type: ignore
 
 
-def parse_status_value(value):
+def parse_status_value(value: Union[str, int]) -> int:
     if value in STATUS_QUERY_CHOICES:
-        return STATUS_QUERY_CHOICES[value]
+        return STATUS_QUERY_CHOICES[value]  # type: ignore[no-any-return]
```

Comment:
This function should return an int. If it doesn't, that's a bug that shouldn't be ignored. If `STATUS_QUERY_CHOICES[value]` is definitely an int, you should cast it to an `int` here instead of ignoring the error.